subtests/docker_cli/create/create_signal.py

```diff
@@ -29,8 +29,8 @@ def run_once(self):
         sigdkrcmd = DockerCmd(self, 'kill',
                               ['--signal', str(sig),
                                self.get_cid()])
-        sigdkrcmd = mustfail(sigdkrcmd.execute(), 1)
-        self.sub_stuff['sigdkrcmd'] = sigdkrcmd
```

Comment:
R0204: 32,8: create_signal.run_once: Redefinition of sigdkrcmd type from dockertest.dockercmd.DockerCmd to autotest.client.shared.utils.CmdResult